client/ayon_core/tools/publisher/control.py

```diff
@@ -2096,7 +2097,7 @@ def get_subset_name(
         if instance_id:
             instance = self.instances[instance_id]
 
-        return creator.get_subset_name(
+        return creator.get_product_name(
```

Comment:
I'm not sure how should be corrected. but, I think there's something in the arguments of this method `creator.get_product_name` call.
As, they don't match with this: 
https://github.com/ynput/ayon-core/blob/8dd9d25ac44ec58e1ca0e7a0214f846d92adf0c1/client/ayon_core/pipeline/create/product_name.py#L82-L93

Reply:
Good point, changed order in the method to look more similar (not exactly the same).